science/gmtsar-data/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                gmtsar-data
+version             1.0
+categories          science
+platforms           darwin
+license             GPL-3
+
+maintainers         ucsd.edu:dsandwell \
+                    ucsd.edu:xix016 \
+                    ucsd.edu:jdesanto \
+                    openmaintainer
+
+supported_archs     noarch
+
+description         Optional orbits for GMTSAR
+long_description    Orbit solutions for the ERS and ENVISAT satellites, required to \
+                    process SAR data from these platforms.
+
+homepage            https://topex.ucsd.edu/gmtsar/
+master_sites        ftp://topex.ucsd.edu/pub/gmtsar/
+
+extract.suffix      .tar
+extract.cmd         tar
+extract.pre_args    -xvf
+
+distname            ORBITS
```

Comment:
Since the `distname` does not contain the version number, you should set `dist_subdir ${name}/${version}`. See https://trac.macports.org/wiki/PortfileRecipes#unversioned-distfiles.